docs/source/user_guide/data_checks.ipynb

```diff
@@ -102,14 +254,14 @@
    },
    "outputs": [],
    "source": [
-    "data = np.random.randn(100, 100)\n",
+    "data = np.tile(np.arange(10) * 0.01, (100, 10))\n",
```

Comment:
Just noticed that in the text above we mention Isolation Forests but we recently changed the implementation to no longer use them.